client/src/app/domain/models/meeting-mediafile/meeting-mediafile.ts

```diff
@@ -0,0 +1,122 @@
+import { Fqid, Id } from '../../definitions/key-types';
+import { HasMeetingId } from '../../interfaces';
+import { HasListOfSpeakersId } from '../../interfaces/has-list-of-speakers-id';
+import { HasProjectionIds } from '../../interfaces/has-projectable-ids';
+import { HasProperties } from '../../interfaces/has-properties';
+import { BaseModel } from '../base/base-model';
+import {
+    FONT_PLACES,
+    FontPlace,
+    LOGO_PLACES,
+    LogoPlace,
+    MediafileMeetingUsageIdKey
+} from '../mediafiles/mediafile.constants';
+
+/**
+ * Representation of MeetingMediaFile
+ * @ignore
+ */
+export class MeetingMediafile extends BaseModel<MeetingMediafile> {
+    public static COLLECTION = `meeting_mediafile`;
+
+    public mediafile_id: Id;
```

Comment:
These non-null assertions in our models are actually wrong because in Typescript this means that these value cannot be null or undefined. But except for the `id` fields this cannot be guaranteed because even properties specified as required can be `undefined` in the client when they are not requested in the autoupdate subscription. 

But this is a topic for another day.